lib/codegen/meta/isa/x86/recipes.py

```diff
@@ -544,6 +544,16 @@ def floatccs(iform):
         sink.put8(imm as u64);
         ''')
 
+# XX /n Unary with floating point immediate.
+# FIXUP: ^ I am unusure about the notation above.
+fimm_z = TailRecipe(
+    'fimm_z', UnaryImm, size=4, ins=(), outs=FPR,
+    instp=IsZero(UnaryImm.imm),
+    emit='''
+        unimplemented!();
+    ''')
```

Comment:
I was a little unsure about where to go from here. Should I be using a `PUT_OP` statement here?

Reply:
Yeah, I think the simplest approach here would be to stay as close to the `fa` recipe as possible.